roles/openshift_facts/library/openshift_facts.py

```diff
@@ -730,6 +730,9 @@ def set_manageiq_facts_if_unset(facts):
             raise OpenShiftFactsInternalError(
                 "Invalid invocation: The required facts are not set"
             )
+    if 'use_manageiq' in facts['common']:
+        if facts['common']['use_manageiq'] is not None:
+            return facts
```

Comment:
I see it as two cases and I was thinking `set_manageiq_facts_if_unset` should handle both.  If `use_manageiq` doesn't even exist in the common facts then it should set it.  If it does exist in the common facts but is set to the default of None then it should be set.

However, thinking more about how we're setting defaults I think we have logic in two places:

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_common/tasks/main.yml#L26

There we set the default to `None` and here our real goal is for the default to be based on `version_greater_than_3_1_or_1_1`.  Is the logic in `openshift_common` actually needed there?